File: trunk/quests/qey2hh1/Tarnic_McWillows.pl
sub EVENT_ITEM {
  if (plugin::check_handin(\%itemcount, 21987 => 1)) {#need correct text
    quest::say("It has been a long time since someone has come for my herbs.  I do still have them however.  I knew that the elves would remember I was out here.  Here is the antidote you require.");
    quest::summonitem(21988);
  }
}

File: trunk/quests/qey2hh1/Tarnic_McWillows.lua
function event_trade(e)
	local item_lib = require("items");

	if(item_lib.check_turn_in(e.trade, {item1 = 21987})) then -- need correct text
		e.self:Say("It has been a long time since someone has come for my herbs.  I do still have them however.  I knew that the elves would remember I was out here.  Here is the antidote you require.");
		e.other:SummonItem(21988);
		e.other:Ding();
		e.other:AddEXP(0);
	end
	item_lib.return_items(e.self, e.other, e.trade)
end
